Castanet/Feed.php

```diff
@@ -142,9 +172,19 @@ public function setManagingEditor($managing_editor)
 	// }}}
 	// {{{ public function setImage()
 
-	public function setImage($url, $width, $height)
+	public function setImage($url)
```

Comment:
Since this sets the iTunes image it may make sense to rename as `setItunesImage()`. The new method can use the `setImage()` name. This will require updating the `Rap` PR.